packages/x-date-pickers/src/internals/hooks/usePicker/usePickerValue.types.ts

```diff
@@ -344,7 +340,31 @@ export interface UsePickerValueContextValue {
    */
   setOpen: React.Dispatch<React.SetStateAction<boolean>>;
   /**
-   * `true` if the picker is open, `false` otherwise.
+   * Set the current value of the picker to be empty.
+   * The value will be `null` on single pickers and `[null, null]` on range pickers.
    */
-  open: boolean;
+  clearValue: () => void;
+  /**
+   * Set the current value of the picker to be the current date.
+   * The value will be `today` on single pickers and `[today, today]` on range pickers.
+   * With `today` being the current date, with its time set to `00:00:00` on Date Pickers and its time set to the current time on Time and Date Pickers.
+   */
+  setValueToToday: () => void;
+  /**
+   * Accept the current value of the picker.
+   * Will call `onAccept` if defined, and if the picker is closed, this value will be retained.
+   */
+  acceptValueChanges: () => void;
+  /**
+   * Cancel the changes made to the current value of the picker.
+   * The value will be reset to the last accepted value.
+   */
+  cancelValueChanges: () => void;
+}
+
+export interface UsePickerValuePrivateContextValue {
+  /**
+   * Closes the picker and accepts the current value if it is not equal to the last accepted value.
+   */
+  dismissViews: () => void;
```

Comment:
I would love to get rid of this method :grimacing: 
It was never used in a public component but was part of the public API since it was passed to `PickersLayout` (:facepalm:), so I'm making it private.